ai/main.py

```
@@ -38,21 +55,50 @@ def synopsys(book_id: int, authorization: str = Header(None)):
+                background = background,
+                choice = request.choice,
+                book_id = book_id)
             return {"status": "success", "result":result}
         else:
             raise HTTPException(status_code=404,detail="데이터가 없습니다.")
     except Exception as e:
         raise HTTPException(status_code=500,detail=f"책 조회 실패: {e}") from e
 
+
+@app.post("/v1/book/{book_id}/story/end")
+def end_story(book_id: int, authorization: str=Header(None)):
```

Flake8 (7.2.0)

[error] missing whitespace around parameter equals

(E252)